lib/health_cards/key.rb

```diff
@@ -16,6 +16,20 @@ def self.enforce_valid_key_type!(obj, allow_nil:  false)
       raise InvalidKeyException.new(self, obj) unless obj.is_a?(self) || (allow_nil && obj.nil?)
     end
 
+    # Create a key from a JWK
+    #
+    # @param jwk_key [Hash] The JWK represented by a Hash
+    # @return [HealthCards::Key] The key represented by the JWK
+    def self.from_jwk(jwk_key)
+      jwk_key = jwk_key.transform_keys(&:to_sym)
+      group = OpenSSL::PKey::EC::Group.new('prime256v1')
```

Comment:
Since we're always using this group, maybe make this a constant.

Reply:
I don't think that is safe unless `Group` is immutable.